### DISCUSSION TITLE
Can not use float values in numeric inputs

### DISCUSSION THREAD

**Discussion by eislambey** [filamentphp/filament#196] [Ideas]
**Describe the bug**
Float values are not accepted on numeric input because default `step` value is 1. Ability to setting step can be nice and will solve the issue.

**To reproduce**
1. Create a TextField and set it numeric
2. Type a float value
3. Click save button

**Expected behavior**
I should be able to write float value on numeric input.

**Screenshots**
![image](https://user-images.githubusercontent.com/10918436/110963340-00010580-8363-11eb-9531-b4d069b8670d.png)

**Context**
- Package version: 1.5.5
- Livewire version: 2.4.0
- Laravel version: 8.32.1
- Browser: Chromium
- Server OS: Arch Linux
- PHP version: 8.0.1

**Additional Details**
- Adding `step="any"` to input can be alternative solution.
- [More info about step attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/step)

**My Code**

```php
public static function form(Form $form): Form
{
    return $form
        ->schema([
            Components\Checkbox::make("is_active"),
            Components\TextInput::make("name"),
            Components\TextInput::make("price")
                ->numeric(),
            Components\TextInput::make("currency"),
        ]);
}
```

**Comment by danharrin** [comment]
Thanks! We will add the step option to the `TextInput` field.

**Comment by eislambey** [comment]
Adding `extraAttributes(["step" => "0.01"])` solved my problem. So it was not actually a bug.

Having step option will be good improvement. Thanks!
